docs/reference/modules/coordination.asciidoc

```diff
@@ -0,0 +1,461 @@
+[[modules-cluster-coordination]]
+== Cluster coordination
+
+The cluster coordination module is responsible for electing a master node and
+managing changes to the cluster state.
+
+[float]
+=== Quorum-based decision making
+
+Electing a master node and changing the cluster state both work robustly by
+using multiple nodes, only considering each action to have succeeded on receipt
+of responses from a majority of the master-eligible nodes in the cluster. The
+advantage of requiring a majority of nodes to respond is that it allows for
+nearly half of the master-eligible nodes to fail without preventing the cluster
+from making progress, but also does not allow the cluster to "split brain",
+i.e. to be partitioned into two pieces each of which may make decisions that
+are inconsistent with those of the other piece.
+
+Elasticsearch allows you to add and remove master-eligible nodes to a running
+cluster. In many cases you can do this simply by starting or stopping the nodes
+as required, as described in more detail below. As nodes are added or removed
+Elasticsearch maintains an optimal level of fault tolerance by updating the
+cluster's _configuration_, which is the set of master-eligible nodes whose
+responses are counted when making decisions such as electing a new master or
+committing a new cluster state. A decision is only made once more than half of
+the nodes _in the configuration_ have responded. Usually the cluster
+configuration is the same as the set of all the master-eligible nodes that are
+currently in the cluster, but in some situations they may be different. As long
+as more than half of the nodes in the configuration are still healthy then the
+cluster can still make progress. The configuration is managed automatically by
+Elasticsearch and stored in the cluster state so you can inspect its current
+contents by (TODO API call?).
```

Comment:
maybe through _cluster/state API?

Reply:
Yep. I would like to include the appropriate `?filter_path=...` argument in the example given here, but I added the API call itself at least.